tests/test_payments.py

```diff
@@ -59,10 +59,20 @@ def test_cancel_payment(client, response):
     assert canceled_payment.id == PAYMENT_ID
 
 
-def test_cancel_payment_invalid_id(client):
+def test_payment_invalid_id(client):
```

Comment:
The other resources contain basically the same tests (checking if `get` or `update` (or any other method) throws an IdentifierError). However, they all do so in separate methods. Could you place the tests for `get` and `update` in separate methods?

Reply:
Fixed.